### PR TITLE
DM-15536: adapt to moving visit definition out of ingest

### PR DESCRIPTION
**Pull request by TallJimbo**
The tests here now just test that trying to define visits from the test data already included in the repo correctly does nothing, because (apparently) none of the exposures here have `observation_type = 'science'`.  When that changes someday in the future (and we include those data IDs in the ingest tests), the tests will fail unless we also set the "visits" attribute to a non-empty dictionary of expected visits, as in the obs_subaru and obs_decam tests.

### CODE REVIEW

File: config/convertRepo.py [hunk 1]
```diff
@@ -10,3 +10,7 @@
 config.datasetIgnorePatterns.extend(["_raw", "raw_hdu", "raw_amp"])
 
 config.curatedCalibrations.extend(["defects", "qe_curve"])
+
+import os
+from lsst.utils import getPackageDir
+config.defineVisits.load(os.path.join(getPackageDir("obs_lsst"), "config", "defineVisits.py"))
```
Comment (timj) [inline config/convertRepo.py:16]: You do not need to use getPackageDir here. Instead you can use the new `__file__` since you are wanting to read the `defineVisits.py` file from the same directory that this config file is located.

Comment (TallJimbo) [reply config/convertRepo.py:16]: `getPackageDir` seems to be a pretty strong convention in obs package config files - especially in obs_subaru, which has many more, but here as well - and I'm wary of going against it without explicitly RFC'ing or at least broadly announcing an intent to move away from it.

Comment (timj) [reply config/convertRepo.py:16]: It's a strong convention because until February (DM-23359) you couldn't use `__file__` -- we had a long discussion on Slack back then where we realized that the same logic that led us to specify test data relative to the tests works for config files where we absolutely want to find the config file in the same directory of another config file. getPackageDir is a hack in that context but it's the only thing we could do previously.

Comment (timj) [reply config/convertRepo.py:16]: Oh and there was a [community post](https://community.lsst.org/t/pex-config-configs-now-know-where-they-are/4023).

Comment (TallJimbo) [reply config/convertRepo.py:16]: I started an across-the-board conversion of all three Gen3 obs packages to use file `__file__`, but I've decided that's enough that it merits its own ticket.  I've put all that on [DM-24495](https://jira.lsstcorp.org/browse/DM-24495), which is now out for review, and if that merges first I'll modify this branch to use the new style.  If it merges second I'll fix the DM-24495 branch.
